phys2bids/phys2bids.py

```diff
@@ -222,6 +222,7 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     elif ftype == 'txt':
         from phys2bids.interfaces.txt import populate_phys_input
     else:
+        # This is never called.
```

Comment:
Any particular reason for this? Is the issue that it's not reachable or that, in practice, no one will actually try running a file without `.txt.` or `.acq`?

Reply:
It is not reachable as [this](https://github.com/physiopy/phys2bids/blob/c20cef844f3b22741635a8d0f2178433c6a548c8/phys2bids/utils.py#L107) exception will be raised before ([see here](https://github.com/physiopy/phys2bids/blob/c20cef844f3b22741635a8d0f2178433c6a548c8/phys2bids/phys2bids.py#L197)) getting to this point.

Reply:
I didn't remove that part in this PR because it's meant to tackle the tests only. We could do it in the MATLAB interface PR if we agree that's the way to go (I think it should be done).

Reply:
Thanks. Yes, it should totally be removed. Plus that's two untested lines being dropped (yay coverage increase!).

Reply:
My latest commit should take care of this. Thanks for the review!